dbcon.js

```diff
@@ -4,6 +4,7 @@ var pool = mysql.createPool({
     user: "cs361_<onid>",
     password: "<password>",
     database: "cs361_<onid>",
+    user: "cs361_frutizg",
```

Comment:
please reset the username to the generic one and add your dbcon to your .gitignore